src/ElmSharp/ElmSharp/AnimationView.cs

```diff
@@ -207,23 +239,173 @@ public double DurationTime
+                return new Size(w, h);
+            }
+        }
+
+        /// <summary>
+        /// Get current animation view state.
+        /// </summary>
+        /// <since_tizen> preview </since_tizen>
+        public AnimationViewState State
+        {
+            get
+            {
+                return (AnimationViewState)Interop.Elementary.elm_animation_view_state_get(Handle);
+            }
+        }
+
+        /// <summary>
+        /// Get the status whether current animation is on playing forward or backward.
+        /// </summary>
         /// <remarks>
-        /// When you required to jump on a certain frame instantly,
-        /// you can change current keyframe by using this property
-        /// The range of keyframe is 0 ~ 1.
+        /// If AnimationView is not on playing, it will return False.
         /// </remarks>
+        /// <since_tizen> preview </since_tizen>
+        public bool IsReversePlaying
+        {
+            get
+            {
+                return Interop.Elementary.elm_animation_view_is_playing_back_get(Handle);
+            }
+        }
+
+        /// <summary>
+        /// Get the index of end frame of the AnimationView, if it's animated.
         /// </summary>
+        /// <remarks>
+        /// Frame number starts with 0.
+        /// </remarks>
+        /// <since_tizen> preview </since_tizen>
+        public int FrameCount
+        {
+            get
+            {
+                return Interop.Elementary.elm_animation_view_frame_count_get(Handle);
+            }
+        }
+
+        /// <summary>
+        /// Sets or Gets the start progress of the play
+        /// </summary>
+        /// <remarks>
+        /// Default value is 0.
+        /// </remarks>
+        /// <since_tizen> preview </since_tizen>
+        public double MinProgress
+        {
+            get
+            {
+                return Interop.Elementary.elm_animation_view_min_progress_get(Handle);
+            }
+            set
+            {
+                Interop.Elementary.elm_animation_view_min_progress_set(Handle, value);
+            }
+        }
+
+        /// <summary>
+        /// Sets or Gets the last progress of the play
+        /// </summary>
+        /// <remarks>
+        /// Default value is 1.
+        /// </remarks>
+        /// <since_tizen> preview </since_tizen>
+        public double MaxProgress
+        {
+            get
+            {
+                return Interop.Elementary.elm_animation_view_max_progress_get(Handle);
+            }
+            set
+            {
+                Interop.Elementary.elm_animation_view_max_progress_set(Handle, value);
+            }
+        }
+
+        /// <summary>
+        /// Sets or Gets the start frame of the play
+        /// </summary>
+        /// <remarks>
+        /// Default value is 0.
+        /// </remarks>
+        /// <since_tizen> preview </since_tizen>
+        public int MinFrame
```

Comment:
```suggestion
        public int FirstFrame
```

Reply:
Lottie is provided APIs for play animation segments such as setMinFrame(), setMinProgress(), and so on. I know that EFL native animation view is implemented according to this lottie lib APIs.
Please refer to page (https://airbnb.io/lottie/android/android.html#sample-app)
I think ElmSharp should follow this APIs name like native EFL and lottie.